using-timescaledb/ingesting-data.md

```diff
@@ -0,0 +1,23 @@
+# Ingesting data
+
+TimescaleDB can support standard SQL inserts. Read more about how to use
+SQL to write data into TimescaleDB in our [Writing Data][writing-data] section.
+
+Users often choose to leverage existing 3rd party tools to build data ingest
+pipelines. Any tools that work with PostgreSQL also work with TimescaleDB.
+
+## Prometheus [](prometheus)
+
+Prometheus is a popular tool used to monitor infrastructure metrics. It can scrape
+any endpoints that expose metrics in a Prometheus-compatible format. Prometheus
+can use TimescaleDB as a remote storage solution to store long-term metrics. Combine TimescaleDB with Prometheus to expand your [querying capabilities][SQL-monitoring]
+by either querying Prometheus using PromQL or querying TimescaleDB using SQL. Queries
+run through Prometheus will automatically query TimescaleDB to surface longer term
+metrics. Read more about how to set up Prometheus with TimescaleDB in our
+[Prometheus tutorial][prometheus-tutorial]. You can also consider using [Grafana][grafana]
+to visualize your metrics.
```

Comment:
"Given our development of a remote storage adaptor and database extension, Prometheus can use TimescaleDB as a remote storage solution to store long-term metrics.  This allows you to combine.."  <-- Maybe cite the github?  Not sure.

"use Grafana or [other tools][<point to our new viz page>] to visualize your metrics."

Reply:
Ah, this is great. Made some edits inspired by this.